src/core/utils/misc.cpp

```diff
@@ -241,6 +242,11 @@ int Utils::Misc::pythonVersion()
         python_proc.start("python3", QStringList() << "--version", QIODevice::ReadOnly);
         if (python_proc.waitForFinished()) {
             if (python_proc.exitCode() == 0) {
+                QByteArray output = python_proc.readAllStandardOutput();
+                if (output.isEmpty())
+                    output = python_proc.readAllStandardError();
+                const QByteArray version_str = output.split(' ').last();
+                Logger::instance()->addMessage(QCoreApplication::translate("misc", "Python version: %1").arg(QString(version_str)), Log::INFO);
                 version = 3;
```

Comment:
@sledgehammer999 I'm not thinking in big changes, just something like that, the same we are doing now but with `python` also:
```
python_proc.start("python", QStringList() << "--version", QIODevice::ReadOnly);
```